.pyre_configuration

```diff
@@ -5,5 +5,6 @@
     "exclude": [
         ".*/\\.tox/.*"
     ],
+    "workers": 3,
```

Comment:
What's the Pyre bug this fixes?

Reply:
It is an issue where workers < 3 causes pyre to SIGSEGV. We discovered the issue (and the workaround) in this PR https://github.com/Instagram/LibCST/pull/579

Reply:
Yikes, thanks for the conext!